config/config.go

```diff
@@ -122,6 +129,14 @@ func Read(targetProfile string, upstreamProfile string, threads int, strExecutor
 		return nil, errors.New(fmt.Sprintf("no models config found, expected to find `models: %s:` in `dbt_project.yml`", project.Name))
 	}
 
+	if seedCfg, found := project.Seeds[project.Name]; found {
```

Comment:
The properties package can already parse Seed schemas (https://github.com/monzo/ddbt/blob/master/properties/propertiesFile.go#L14)

I wonder if we should move the whole `dbt_project.yml` parsing over to the properties package too?